src/pydata_sphinx_theme/assets/styles/variables/_color.scss

```diff
@@ -68,17 +93,31 @@ html[data-theme="light"] {
 */
 html[data-theme="dark"] {
   /*****************************************************************************
-  * main colors
+  * main colors. See notes under data-theme="light", above.
   */
-  --pst-color-primary: rgb(69 157 185);
-  --pst-color-secondary: rgb(238 144 64);
-  --pst-color-success: rgb(72, 135, 87);
-  --pst-color-info: var(--pst-color-primary);
-  --pst-color-warning: var(--pst-color-secondary);
-  --pst-color-danger: rgb(203, 70, 83);
-  --pst-color-text-base: rgb(201, 209, 217);
-  --pst-color-text-muted: rgb(192, 192, 192);
-  --pst-color-border: rgb(192, 192, 192);
+  $pst-semantic-colors: (
```

Comment:
Could we somehow make this a dictionary-like object at the top of the file, with `[light]` and `[dark]` keys? Then we can access the colors via those keys in our SCSS logic below? Might make it easier to keep everything in one place.